backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/model/validator/M_HU.java

```diff
@@ -83,6 +83,23 @@ public void validateStatusChange(@NonNull final I_M_HU hu)
 		huStatusBL.assertStatusChangeIsAllowed(hu, oldHu.getHUStatus(), hu.getHUStatus());
 	}
 
+	@ModelChange(timings = ModelValidator.TYPE_BEFORE_CHANGE, ifColumnsChanged = I_M_HU.COLUMNNAME_HUStatus)
+	public void handleHUUniqueAttributes(@NonNull final I_M_HU hu)
+	{
+		final IHUStatusBL huStatusBL = Services.get(IHUStatusBL.class);
+		final HUUniqueAttributesService huUniqueAttributesService = SpringContextHolder.instance.getBean(HUUniqueAttributesService.class);
```

Comment:
please inject the service...either via `backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/model/validator/Main.java`
or by turning it into a `@Component`